src/tasks/SwiftProcess.ts

```diff
@@ -72,60 +122,26 @@ export class SwiftProcess {
         }
     }
 
-    /**
-     * Write a VT sequence as a string to the pty process
-     * to use as stdin
-     *
-     * @param s string to write to pty
-     */
     handleInput(s: string): void {
         this.spawnedProcess?.write(s);
     }
 
-    /**
-     * Forcefully kill the pty process. The {@link onDidClose}
-     * event will fire with exit code 8
-     */
-    kill(): void {
+    kill(signal?: NodeJS.Signals): void {
         if (!this.spawnedProcess) {
             return;
         }
-        this.spawnedProcess.kill();
-        this.closeEmitter.fire(8);
+        this.spawnedProcess.kill(signal);
     }
 
-    /**
-     * Resize the pty to match the new {@link vscode.Pseudoterminal} dimensions
-     *
-     * @param dimensions
-     */
```

Comment:
How come you are deleting all the comments?

Reply:
I moved them to the interface. They're still shown in vscode on hover over the implementation

Reply:
Fair enough